text-objects.kak

```diff
@@ -77,13 +82,13 @@ define-command -hidden text-object-indented-paragraph %{
 # depends on occivink/vertical-selection.kak
 define-command -hidden text-object-vertical %{
   evaluate-commands %sh{
-    case "$kak_opt_last_mode" in
-      '<a-i>') k='<esc>:select-vertically<ret>' ;;
-      '<a-a>') k='<a-i>w<esc>:select-vertically<ret>' ;;
-      '[') k='<esc>:select-up<ret>' ;;
-      ']') k='<esc>:select-down<ret>' ;;
-      '{') k='<a-i>w<esc>:select-up<ret>' ;;
-      '}') k='<a-i>w<esc>:select-down<ret>' ;;
+    case "$kak_opt_objects_last_mode" in
+      '<a-i>') k='<esc>:<space>select-vertically<ret>' ;;
+      '<a-a>') k='<a-i>w<esc>:<space>select-vertically<ret>' ;;
+      '[') k='<esc>:<space>select-up<ret>' ;;
+      ']') k='<esc>:<space>select-down<ret>' ;;
+      '{') k='<a-i>w<esc>:<space>select-up<ret>' ;;
+      '}') k='<a-i>w<esc>:<space>select-down<ret>' ;;
```

Comment:
Since there's no way to use space directly, but I want to strip all commands form commandline history, I used `<space>` as key in `execute-keys` command. It works just fine.